docs/paper/paper.bib

```diff
@@ -251,3 +251,28 @@ @article{liu_pra_2019
   year    = {2019},
   doi     = {10.1103/PhysRevA.100.052511}
 }
+
+% scuff-em
+% https://doi.org/10.1109/TAP.2015.2438393
+@article{reid_ieee_2015,
+  author  = {Reid, M. T. H. and Johnson, S. G.},
+  title   = {{Efficient Computation of Power, Force, and Torque in BEM Scattering Calculations}},
+  journal = {IEEE Trans. Antennas Propag.},
+  volume  = {63},
+  issue   = {8},
+  pages   = {3588--3598},
+  year    = {2015},
+  doi     = {10.1109/TAP.2015.2438393}
+}
+
+% meep
+@article{oskooi_cpc_2010,
+  author  = {Oskooi, A. and Roundy, D. and Ibanescu, M. and Bermel, P. and Joannopoulos, J. D. and Johnson, S. G.},
+  title   = {{Meep: A flexible free-software package for electromagnetic simulations by the FDTD method}},
+  journal = {Comput. Computer Physics Communications},
```

Comment:
"Comput." is doubling "Computer", the title could be abbreviated as "Comp. Phys. Comm."